save-core/build.gradle.kts

```diff
@@ -16,15 +15,11 @@ kotlin {
             }
         }
     }
-    val os = getCurrentOperatingSystem()
-    // Create a target for the host platform.
-    val hostTarget = when {
```

Comment:
This should be changed for plugins as well, it affects all artifacts that we are deploying

Reply:
Fixed

Reply:
And `warn-plugin`?

Reply:
Done